build.assets/Makefile

```diff
@@ -11,13 +11,17 @@ NOROOT=-u $$(id -u):$$(id -g)
 KUBECONFIG ?=
 TEST_KUBE ?=
 
+SHELL=bash
```

Comment:
The `busybox` that `docker:git` and similar images for Drone use only has busybox `sh` out of the box - you can install `bash` with `apk --no-cache add bash` but why do we need it?

Reply:
I already removed it for lack of a better thought. The reason I added it initially is that I could not build specific `release` targets as these are bash-flavored so it was the knee jerk action.